README.md

```diff
@@ -18,14 +18,20 @@ are unit tested and should be working as long as the build status is passing.
 ### As user
 
 ```
-LSTCHAIN_VER=0.10.0  (or the version you want to install - usually the latest release)
+LSTCHAIN_VER=0.10.3  (or the version you want to install - usually the latest release)
 wget https://raw.githubusercontent.com/cta-observatory/cta-lstchain/v$LSTCHAIN_VER/environment.yml
 conda env create -n lst -f environment.yml
 conda activate lst
 pip install lstchain==$LSTCHAIN_VER
 rm environment.yml
 ```
 
+Alternatively, you can directly create an environment and install lstchain from conda-forge as:
```

Comment:
For *users*, I would put this simpler instruction first.

Reply:
done